cfme/tests/ansible/test_embedded_ansible_basic.py

```diff
@@ -214,6 +214,24 @@ def test_embedded_ansible_repository_invalid_url_crud(request, appliance, wait_f
     repository.delete_if_exists()
 
 
+@pytest.mark.rhel_testing
+@pytest.mark.tier(1)
+def test_embedded_ansible_private_repository_crud(request, ansible_private_repository):
+    """
+    Add Private Repository by using SCM credentials. Check repository gets added successfully.
+
+    Polarion:
+        assignee: sbulage
+        casecomponent: Ansible
+        caseimportance: critical
+        initialEstimate: 1/6h
+        tags: ansible_embed
+    """
+    assert ansible_private_repository.exists
```

Comment:
Good start for the test, is there no updating a private repo? I'd like to see some more automation around the user behaviors concerning these repos - how MIQ behaves when the repo is modified and becomes unreadable, or whether a change to the repo URL successfully updates the contents for embedded automation.